app/models/feedback_reporters/feedback_reporter.rb

```diff
@@ -24,35 +24,22 @@ def description
   end
 
   def send_report!
-    # We're sending the XML data via a URL to our Support ticket service. The URL needs to be Percent-encoded so that
-    # everything shows up correctly on the other end. (https://en.wikipedia.org/wiki/Percent-encoding)
+    # We're sending the XML data via a URL to our Support ticket service. The
+    # URL needs to be Percent-encoded so that everything shows up correctly on
+    # the other end. (https://en.wikipedia.org/wiki/Percent-encoding)
     encoded_xml = CGI.escape(xml.to_str)
-    HTTParty.post("#{ArchiveConfig.BUGS_SITE}",
-      body: "&xml=#{encoded_xml}"
-    )
-  end
-
-  def send_abuse_report!
-    HTTParty.post("#{ArchiveConfig.ABUSE_REPORTS_SITE}/projects/#{project_id}/bugs",
-      headers: {
-        "Content-Type" => "application/xml",
-        "Accept" => "application/xml"
-      },
-      basic_auth: {
-        username: ArchiveConfig.ABUSE_REPORTS_USER,
-        password: ArchiveConfig.ABUSE_REPORTS_PASSWORD
-      },
-      body: xml
-    )
+    puts "ORANGEZ: #{ArchiveConfig.BUGS_SITE}/#{project_path}#{encoded_xml}"
+    HTTParty.post("#{ArchiveConfig.BUGS_SITE}#{project_path}".to_s,
```

Comment:
I don't think this needs `.to_s` anymore -- iirc, you originally added it just because Hound wanted you to switch from the string interpolation method of `#{ArchiveConfig.BUGS_SITE}` to the prettier `ArchiveConfig.BUGS_SITE.to_s`